lib/Controller/LoginController.php

```diff
@@ -15,6 +15,7 @@
 use OCP\ISession;
 use OC\User\LoginException;
 use OCA\SocialLogin\Storage\SessionStorage;
+use OCA\SocialLogin\Provider\CustomOAuth2Connect;
```

Comment:
Should use `CustomOAuth2` name

Reply:
Why didn't you call the `CustomOpenIDConnect` then `CustomOpenID` ?

Reply:
OpenID connect is NOT OpenID 😉  
http://openid.net/connect/